djangocms_bootstrap4/contrib/bootstrap4_utilities/models.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+
+from cms.models import CMSPlugin
+
+from djangocms_bootstrap4.fields import (
```

Comment:
This is ok (IMHO) but just fyi, no need to break it into multiple lines since is less than 79 chars